articles/desktop-flows/virtual-desktops.md

```diff
@@ -14,117 +14,97 @@ search.audienceType:
 
-If the versions aren't synced during recording or UI element capturing, a dialog will prompt you to sync the versions. In this case, select **Sync** to sync the two products. 
+:::image type="content" source="media\virtual-desktops\virtual-desktops-agent-sync-dialog.png" alt-text="Screenshot of the prompt to sync Power Automate and Power Automate agent for virtual desktops.":::
 
-  ![Screenshot of a dialog prompting you to sync Power Automate for desktop and the agent for virtual desktops.](media\virtual-desktops\virtual-desktops-agent-sync-dialog.png)
-
-If the versions are out of sync during the runtime of a desktop flow, Power Automate for desktop will sync the versions of the two products automatically.
+If the versions are out of sync when a desktop flow is running, Power Automate will sync them automatically.
 
 > [!NOTE]
-> The DLLs of the synced version used by the agent can be found in **C:\Users\\[Username]]\AppData\Local\Microsoft\Power Automate Desktop\RDP Automation Agents**.
+> The DLLs of the synced agent are stored in **C:\Users\\*username*\AppData\Local\Microsoft\Power Automate Desktop\RDP Automation Agents**.
 
 ## Distinguish UI elements captured on virtual desktops
 
-Capturing UI elements on virtual desktops works the same way as on physical machines. The generated selectors of windows and UI elements are the same, independently of whether they've been captured on a host machine or a virtual desktop. You can find more information regarding UI elements and selectors in [Automate using UI elements](ui-elements.md).
-
-To distinguish the UI elements captured on virtual desktops, check the visual indications and the tree structure in the UI elements pane.
-
-UI elements captured in the host machine are located under the **Computer** parent, while the UI elements captured on a virtual desktop are located under an **RDP** parent.
+Generated selectors of windows and UI elements are the same regardless of whether they were captured on a physical device or a virtual desktop. Visual indications and the tree structure in the UI elements pane help you to distinguish UI elements captured on virtual desktops from those captured on a physical device. [Learn more about UI elements and selectors](ui-elements.md).
 
-Every individual virtual desktop has its own tree, while numeric prefixes allow you to distinguish virtual desktops of the same type.
+UI elements captured on the physical device are located under the **Computer** parent. UI elements captured on a virtual desktop are located under an **RDP** parent. Every virtual desktop has its own individual tree. A numeric prefix helps you to distinguish virtual desktops of the same type.
+<!--NOTE: Would the parent be called "Citrix" if that was the kind of virtual desktop it was on?-->
```

Comment:
@rhanajoy Currently, there isn't a "Citrix" parent, but it will be added in the feature.

Reply:
Got it! That might be a good note to add, because I wasn't sure if the "RDP" parent was generic for all virtual desktops.

Reply:
For now, we can leave it like that, and I'll talk with the PMs if we need to add a note about it.

Reply:
Sounds good to me!